pkg/sidecar/sidecar_handler.go

```diff
@@ -24,8 +24,9 @@ func handler(ctx context.Context, instance *Instance) error {
 
 	// Network configuration loop.
 	err := instance.Network.ConfigureNetwork(ctx, &network.Config{
-		Network: defaultDataNetwork,
-		Enable:  true,
+		Network:      defaultDataNetwork,
+		Enable:       true,
+		WhitelistAll: true,
```

Comment:
I believe this is changing the behaviour we have until now: so far we did not have `WhitelistAll` for test plans (therefore I added the hack on `oni`, without an intention of it going to `master`).

I think we should set it by default to `false`, and have test plans be explicit when they want to `WhitelistAll`. This however makes it more difficult as routes are initially blocked, and only later allowed, but worth confirming.